test/core/originalNodeNameTests/NestedNodeCustomNode.dyf

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
This file does not seem to be used.

Reply:
@mmisol If you search for the guid you can see there is a dependency on this dyf

Reply:
Sorry, I assumed this file needed to be referenced, by name, somewhere in the tests. Just to satisfy my curiosity, could you explain how this file is being used? Thanks

Reply:
@mmisol 
Sure You can see the origianNodeNameCustomNode.dyn has a property called
```Json
  "Dependencies": [
    "6f65de85-d71f-46e1-b995-dfdeba2adbe3"
  ],
```
This guid is the same guid of this dyf, so you can tell that DYN is actually referencing this dyf. placed this custom node and renamed this node